<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-2</y>
      <width>1500</width>
      <height>1500</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="widgetResizable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>3000</width>
       <height>3000</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>3000</width>
       <height>3000</height>
      </size>
     </property>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>3000</width>
        <height>3000</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveToEdit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionReform"/>
    <addaction name="actionNoRegretMode"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>350</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QLabel" name="labelPenType">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>笔：</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelPenColor">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>画笔颜色</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelColor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>颜色：</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelbackground">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>背景色</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButtonSub">
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="myphotos.qrc">
          <normaloff>:/new/sub.png</normaloff>:/new/sub.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="SliderWidth">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>22</height>
         </size>
        </property>
        <property name="minimum">
         <number>3</number>
        </property>
        <property name="maximum">
         <number>17</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonAdd">
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="myphotos.qrc">
          <normaloff>:/new/add.png</normaloff>:/new/add.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButtonColor">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>210</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonPen">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="myphotos.qrc">
       <normaloff>:/new/pen.png</normaloff>:/new/pen.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonEraser">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="myphotos.qrc">
       <normaloff>:/new/eraser.png</normaloff>:/new/eraser.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="insertImg">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>430</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="myphotos.qrc">
       <normaloff>:/new/thePhoto.png</normaloff>:/new/thePhoto.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="textButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>430</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="myphotos.qrc">
       <normaloff>:/new/theText.png</normaloff>:/new/theText.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonBackgroundColor">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>170</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelShape">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>50</width>
       <height>29</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>形状：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonPenPath">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>300</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="myphotos.qrc">
       <normaloff>:/new/thePenPath.png</normaloff>:/new/thePenPath.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonLine">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>300</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="myphotos.qrc">
       <normaloff>:/new/theLine.png</normaloff>:/new/theLine.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRect">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>300</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="myphotos.qrc">
       <normaloff>:/new/theRect.png</normaloff>:/new/theRect.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonCircle">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>300</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="myphotos.qrc">
       <normaloff>:/new/theCircle.png</normaloff>:/new/theCircle.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonEllipse">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>340</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="myphotos.qrc">
       <normaloff>:/new/theEllipse.png</normaloff>:/new/theEllipse.png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="labelInsert">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>插入：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>470</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>字体设置</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonTextBold">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>490</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="myphotos.qrc">
       <normaloff>:/new/BoldFalse.png</normaloff>:/new/BoldFalse.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonTextItatic">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>490</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="myphotos.qrc">
       <normaloff>:/new/ItaticFalse.png</normaloff>:/new/ItaticFalse.png</iconset>
     </property>
    </widget>
    <widget class="QSlider" name="textSize">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>530</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>myicons/new.png</normaloff>myicons/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>新建</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="myphotos.qrc">
     <normaloff>:/new/new.png</normaloff>:/new/new.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>SaveAsPng</string>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveToEdit">
   <property name="text">
    <string>SaveToEdit</string>
   </property>
   <property name="toolTip">
    <string>另存为</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionReform">
   <property name="text">
    <string>Reform</string>
   </property>
   <property name="toolTip">
    <string>恢复</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>关于</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionNoRegretMode">
   <property name="text">
    <string>NoRegretMode</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myphotos.qrc"/>
 </resources>
 <connections/>
</ui>
